guacamole/pom.xml

```diff
@@ -388,9 +388,14 @@
         <dependency>
             <groupId>org.webjars.bower</groupId>
             <artifactId>angular-translate-loader-static-files</artifactId>
-            <version>2.8.0</version>
+            <version>2.16.0</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>angular-toArrayFilter</artifactId>
```

Comment:
The `orderBy` filter no longer supports objects, which necessitates transforming any object into an array before using it - that's what this new dependency is for.

Reply:
The licenses of bundled dependencies will need to be updated (`guacamole/src/licenses`).

Reply:
Same for the bundled dependencies which are already documented but whose versions have changed:

https://github.com/apache/guacamole-client/blob/a6579ce4e1c70ea436fa1acc26a9ce2bb1e7fe84/guacamole/src/licenses/LICENSE#L214-L220

Reply:
Alright, updated.